tool/gravity/cli/register.go

```diff
@@ -723,7 +723,7 @@ func RegisterCommands(app *kingpin.Application) *Application {
 	g.SystemServiceJournalCmd.Args = g.SystemServiceJournalCmd.Arg("arg", "optional arguments to the journalctl").Strings()
 
 	g.SystemReportCmd.CmdClause = g.SystemCmd.Command("report", "collect system diagnostics and output as gzipped tarball to terminal").Hidden()
-	g.SystemReportCmd.Filter = g.SystemReportCmd.Flag("filter", "collect only specific diagnostics ('system', 'kubernetes'). Collect everything if unspecified").Strings()
+	g.SystemReportCmd.Filter = g.SystemReportCmd.Flag("filter", "collect only specific diagnostics ('system', 'kubernetes', 'etc', 'timeline', 'resources'). Collect everything if unspecified").Strings()
```

Comment:
nit: we should list these in a variable that is updated whenever a new report filter type is added.